.github/workflows/build-test-image.yaml

```
@@ -114,9 +122,38 @@ jobs:
           environment-type: qa
 
-  tests:
+  create-custom-networks:
     needs: omnistrate-update-plans
     runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Create Omnistrate Custom Network - GCP
+        uses: ./.github/actions/create-omnistrate-custom-network
```

actionlint

file "/bin/sh" does not exist in "/cache/48/53cd2386-8703-4aeb-9f37-2c5c197b07ab/home/jailuser/git/.github/actions/create-omnistrate-custom-network". it is specified at "entrypoint" key in "runs" section in "Create Omnistrate Custom Network" action

(action)